spec/requests/users_controller_spec.rb

```diff
@@ -2020,6 +2020,76 @@ def post_user
     end
   end
 
+  describe '#ignore' do
+    it 'raises an error when not logged in' do
+      put "/u/ignore.json"
```

Comment:
Is that the right endpoint?

Reply:
Ah this is bad. I totally forgot to add these two controller methods behind `requires_login` 👀 
Specs to the rescue!